auth/roles-metadata.hh

```diff
@@ -23,19 +23,19 @@ class untyped_result_set_row;
 
 namespace auth {
 
-namespace meta {
-
-namespace roles_table {
+namespace meta::roles_table {
 
 std::string_view creation_query();
 
-constexpr std::string_view name{"roles", 5};
+constexpr std::string_view name = "roles";
 
-constexpr std::string_view role_col_name{"role", 4};
+constexpr std::string_view role_col_name = "role";
+constexpr std::string_view can_login_col_name = "can_login";
+constexpr std::string_view is_superuser_col_name = "is_superuser";
+constexpr std::string_view member_of_col_name = "member_of";
```

Comment:
I am not sure if it's worth to introduce those consts,
1. variable name is longer than original string
2. code is less readable
3. it's not used in all places so anyway it's useless from ref. point of view

Reply:
Agreed, thank you

Reply:
I recommend including the motivation in the commit description for stylistic changes like this. If you are able to come up with a convincing argument in their favor, then it increases your chances for them to be accepted by reviewers. If you can't come up with any, then it's probably better to drop them.

Reply:
I could argue that it makes the code easier to modify, but Marcin's arguments make sense